test/sync/build.gradle.kts

```diff
@@ -22,6 +22,7 @@ plugins {
     id("com.android.library")
     id("io.realm.kotlin")
     id("realm-lint")
+    kotlin("plugin.serialization") version "1.5.21"
```

Comment:
Would it be possible to move this to our Config file in buildSrc?

Reply:
Assuming you just referred to the version number!? Yeah, seems like we can now actually put `Versions` references here 🎉 ... must have been fixed in Gradle.